src/devices/iluminize.ts

```diff
@@ -198,6 +198,22 @@ const definitions: DefinitionWithExtend[] = [
         ],
         meta: {multiEndpoint: true},
     },
+    {
+        zigbeeModel: ['5130.54'],
+        model: '5130.54',
+        vendor: 'Iluminize',
+        description: 'Zigbee handheld remote RGBW 4 channels',
+        extend: [
+            deviceEndpoints({endpoints: {ep1: 1, ep2: 2, ep3: 3, ep4: 4}}),
+            battery(),
+            identify(),
+            commandsOnOff(),
```

Comment:
```suggestion
            commandsOnOff({endpointNames: ['1', '2', '3', '4']}),
```

Please add the endpoints here (for all commands)